openmrs/apps/clinical/dashboard.json

```diff
@@ -10,6 +10,7 @@
                 "displayOrder":0,
                 "ageLimit":100,
                 "patientAttributes":[],
+                "additionalPatientIdentifiers": [],
```

Comment:
You could have added ABHA address and ABHA Number

Reply:
Added "ABHA Address","ABHA Number" as values to the field additionalPatientIdentifiers.